idx/memory/tag_query_test.go

```diff
@@ -286,11 +260,9 @@ func TestGetByTag(t *testing.T) {
 		if len(res) != len(tc.expectation) {
 			t.Fatalf("Result does not match expectation for expressions %+v\nGot:\n%+v\nExpected:\n%+v\n", tc.expressions, res, tc.expectation)
```

Comment:
please expected first, got last

Reply:
like last time, i was commenting on the Fatalf line ;-)

Reply:
lol, right, sorry https://github.com/grafana/metrictank/pull/750/commits/5deda8aea2603aa694abc9009aeb26b881bcddaf